src/azure-cli/azure/cli/command_modules/security/_params.py

```diff
@@ -73,6 +74,15 @@ def load_arguments(self, _):
                 validator=validate_auto_provisioning_toggle,
                 arg_type=auto_provisioning_auto_provision_arg_type)
 
+    for scope in ['atp show']:
+        with self.argument_context('security {}'.format(scope)) as c:
+            c.argument('storage_account_name', help='Name of an existing storage account.')
+
+    for scope in ['atp update']:
+        with self.argument_context('security {}'.format(scope)) as c:
+            c.argument('storage_account_name', help='Name of an existing storage account.')
+            c.argument('is_enabled', help='Enable or disable Advanced Threat Protection for a received storage account. Possible values: {true, false}')
```

Comment:
How about using `get_three_state_flag` for this param? https://github.com/Azure/azure-cli/blob/d1613b0d235327c766e448f89362526e2c15fb90/src/azure-cli-core/azure/cli/core/commands/parameters.py#L160